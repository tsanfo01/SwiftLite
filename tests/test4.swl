//succ test 4
enum Animal {
    case Dawg;
    case Cat;
    case Bird;
}
var a:Animal = Animal.Dawg;