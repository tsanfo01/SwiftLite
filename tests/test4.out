enum Animal{
case Dawg;
case Cat;
case Bird
}
var a: Animal = Animal.Dawg;